collector/mdadm_linux.go

```diff
@@ -76,6 +77,26 @@ func evalStatusline(statusline string) (active, total, size int64, err error) {
 	return active, total, size, nil
 }
 
+func evalRaid0line(statusline string) (size int64, err error) {
+	matches := raid0lineRE.FindStringSubmatch(statusline)
+
+	// +1 to make it more obvious that the whole string containing the info is also returned as matches[0].
+	if len(matches) < 1+1 {
+		return 0, fmt.Errorf("too few matches found in statusline: %s", statusline)
```

Comment:
That a regular expression is used to parse the status line is an implementation detail and this kind of error message doesn't add a value. This whole block can be simplified to:

```go
if len(matches) != 2 {
  return 0, fmt.Errorf("invalid raid0 status line: %s", statusLine)
}
```